llvm/lib/Analysis/Loads.cpp

```diff
@@ -319,11 +319,14 @@ bool llvm::isDereferenceableAndAlignedInLoop(
   const SCEV *MaxBECount =
       Predicates ? SE.getPredicatedConstantMaxBackedgeTakenCount(L, *Predicates)
                  : SE.getConstantMaxBackedgeTakenCount(L);
+  const SCEV *SymbolicMaxBECount =
```

Comment:
This value is identical to `MaxBECount` so why not just pass `MaxBECount` in as both arguments to `getStartAndEndForAccess`?

Reply:
Updated the naming to clarify the names, thanks